REQUIRE

```diff
@@ -11,6 +11,7 @@ MacroTools
 StatsFuns 0.7.0
 SpecialFunctions
 Bijectors
+FDM
```

Comment:
Move the dependency on `FDM` to `test/REQUIRE`?

Reply:
Well, that's what I thought I did! Thanks for pointing this out.